facefusion/json.py

```diff
@@ -0,0 +1,26 @@
+import json
+from typing import Any, Dict, Optional
+
+from facefusion.filesystem import is_file
+
+
+def is_json(json_path : str) -> bool:
+	if is_file(json_path):
+		try:
+			with open(json_path, 'r') as file:
+				json.load(file)
+			return True
+		except json.JSONDecodeError:
```

Comment:
That exception handling belongs into the read_json() method and is_json just uses read_json()